recipes/cvloop/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "cvloop" %}
+{% set version = "0.3.2" %}
+{% set sha256 = "6c6e92420c99a938ffc271b0dba549814bdd13c5821e34eefe91f0bed469bc69" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install
+  skip: True  # [py<36]
+
+requirements:
+  build:
+    - python 3.6.1
+    - opencv 3.2.0
+    - numpy 1.12.0
+    - matplotlib 2.0.0
+    - jupyter_core 4.3.0
+    - jupyter 1.0.0
+  run:
+    - python 3.6.1
+    - opencv 3.2.0
+    - numpy 1.12.0
+    - matplotlib 2.0.0
+    - jupyter_core 4.3.0
+    - jupyter 1.0.0
```

Comment:
Can I set lower bounds just as I do in requirements files, i.e. `>=2.0.0` ? I will try this.

I haven't tried NumPy 1.11 as I use brew for it, which always provides me with the latest version. But it should work, so I will try that. I know that the CI had an issue with 1.9 which seemed the default (see https://circleci.com/gh/conda-forge/staged-recipes/16109):
```
The following specifications were found to be in conflict:
  - numpy 1.9*
  - python 3.6*```